features/step_definitions/book_steps.rb

```diff
@@ -0,0 +1,67 @@
+# require "pry-byebug"
+Before do
+  visit("/users/sign_up")
```

Comment:
Lines 3-9 can be put into a "make account" method that is included as a common step and then called.